packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -47,6 +49,32 @@ interface TokensListProps {
 }
 
 const getTokensByTheme = (theme: ThemeVariants, props: TokensListProps): TokenCategory[] => {
+  const font = props.consoleTokens[0].node.tokens.find(ele => ele.categoryName === 'fonts');
+  if (font) {
+    font.info = (
+      <Callout variant="secondary">
+        <CalloutTitle as="h4">Heads up about fonts in Paste!</CalloutTitle>
+        <CalloutText>
+          Due to font-family licensing, we have no ability to redistribute certain themed fonts. Paste makes the
+          assumption that the Whitney fonts are loaded if you are using the Console theme, and Colfax for Sendgrid.
+        </CalloutText>
+      </Callout>
+    );
+  }
+
+  const fontSize = props.consoleTokens[0].node.tokens.find(ele => ele.categoryName === 'font-sizes');
+  if (fontSize) {
+    fontSize.info = (
+      <Callout variant="secondary">
+        <CalloutTitle as="h4">Heads up about font sizes in Paste!</CalloutTitle>
+        <CalloutText>
+          Fonts are styled using REM (Relative EM{"'"}s). They are relative to the root {'<html>'} element font size. We
+          expect a font size of 16px html.
```

Comment:
Paste expects a base font size of 16px to render text as intended.

Reply:
Happy to change it but want to keep html in there otherwise it’s ambiguous with body

Reply:
"Paste expects the html element to have a font-size of 16px"?